src/cata_utility.h

```diff
@@ -0,0 +1,51 @@
+#ifndef CATA_UTILITY_H
+#define CATA_UTILITY_H
+
+#include "enums.h"
+#include "creature.h"
+#include "item.h"
```

Comment:
Good point, but nc_color is declared as ```typedef int nc_color``` what would the forward decl of that be?